src/AutoRest.CSharp/Common/Output/Models/Shared/Parameter.cs

```diff
@@ -228,6 +230,14 @@ static string AddAllowedValues(string description, ICollection<ChoiceValue> choi
 
         private static Constant? ParseConstant(RequestParameter parameter, TypeFactory typeFactory)
         {
+            if (parameter.In == HttpParameterIn.Header && RequestHeader.ClientRequestIdHeaders.Contains(parameter.Language.Default.SerializedName ?? parameter.Language.Default.Name))
```

Comment:
Is there a case when `SerializedName` is empty?

Reply:
It is rare, but possible.